x/gamm/pool-models/stableswap/pool.go

```diff
@@ -403,9 +404,9 @@ func validateScalingFactors(scalingFactors []uint64, numAssets int) error {
 }
 
 func validatePoolAssets(initialAssets sdk.Coins, scalingFactors []uint64) error {
-	if len(initialAssets) < types.MinPoolAssets {
+	if len(initialAssets) < swaproutertypes.MinPoolAssets {
 		return types.ErrTooFewPoolAssets
-	} else if len(initialAssets) > types.MaxPoolAssets {
+	} else if len(initialAssets) > swaproutertypes.MaxPoolAssets {
 		return types.ErrTooManyPoolAssets
 	}
```

Comment:
Agreed this is to be addressed in a follow-up issue

Reply:
The reason why this hasn't been moved right now is because I'm limiting the scope to only doing the absolutely necessary lifts for the refactor.

Otherwise, this PR would get out of control, reaching a few thousand more LOC. Sorry if this was confusing